### ISSUE TITLE
Review education.prod.drupal.uiowa.edu from a content and design perspective

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#4281]
# Review process

Work with @ABoston88 and put together a two-column spreadsheet.  In the first column provide the page name and url and in the second column provide notes/comments/suggestions. 

## Example 
<table border="0">
 <tr>
    <td><b style="font-size:30px">Page</b></td>
    <td><b style="font-size:30px">Notes</b></td>
 </tr>
 <tr>
    <td>Areas of Study: https://education.prod.drupal.uiowa.edu/areas-study</td>
    <td>
- Add a "Find your major search" section like on admissions.uiowa.edu?
- Consider adding background color to CTA to create visual separation from card rows<br />
- Consider shortening call to action header to something like "Search Programs"<br />
- Is "Programs A-Z" section necessary with the program finder?  It creates more visual clutter on a content heavy page. 
- Consider alternative layout for "Continuing Education section".  A single text area with heading, text, and buttons will be more compact, cohesive, and readable. <br />
- "Online Programs" CTA is pretty crushed in the "Narrow" container, consider using "Normal" for this CTA to avoid multiple lines. <br />
- Is the "Programs by Degree Type" section necessary with the program finder? <br />
- Is the "Programs by Degree Department" section necessary with the program finder? 
    </td>
 </tr>
</table>

Look through every page on the site and review pages using these criteria:

# Menus

- [x] Check for usage of horizontal menu
  - Make sure background color usage is consistent when this is used throughout the site
  - Make sure the vertical spacing looks like the usage on https://uiowa.edu/admissions
  - Check how the horizontal menu looks on all breakpoints
  - Check how it looks and mobile and verify that it has left/right padding
- [x] Main navigation
  - Check all breakpoints on the main navigation and recommend shortening the number of links if they are overflowing the page.

# Blocks 

- [x] How many custom blocks are there on a page? Using more than 2-3 custom block types per page layout might create visual confusion/overload.  
- [x] Check for CTAs using "edge to edge" and recommend using the "normal" option for users on larger screens.
- [x] Check for only buttons being created using the CTA and recommend they use the WYSIWYG button option in the text area. 
- [x] Check for text in containers using background colors without padding.  Offer recommendations to add padding. 
- [x] Check for usages of ALL CAPS on pages (Might be used in block component fields, mostly in banner heading fields or link fields because those fields use CSS to make the text appear in all caps)
  -  Read more about all caps usage here: `https://github.com/humanmade/hm-pattern-library/issues/75#issuecomment-235672885`
- [x] Check for stat component being used for page title/text areas instead of numbers.  
  - Readability is reduced with all caps because the shape of the letters become uniform and make recognizing them harder to read
  - Stat component is not optimized for mobile if being used as a page title
- [x] Cards: If multiple cards are used in the same section, verify that they are all using the same card style for consistency. 
- [x] Banners: check for `<h>`tags used in the banner description.   `<h5>` or `<h6>` tags are often used to create bolder banner description. 

# Background colors
- If black and gold background colors/patterns are used liberally, make sure it is balanced ample negative space for readability.  Offer recommendations to reduce the amount of background colors and patterns if page appears too busy.  Usually, one section using a background pattern/color paired with white or gray sections is recommended. 
- For most sites, this guidance from brand.uiowa.edu is probably the best advice: "**Keep it neutral by leaning more on black, grey, and white, but leverage gold to add contrast or accent key pieces of information."**

# Layouts

- [x] Rows: check for consistent use of similar blocks in layout rows.  A best practice example is a 3 or 4 column row of cards.  Check if layout rows are using consistent block components in rows and recommend consistency if you see a visually jarring combination like card, banner, stat in a 3-column or 4-column row.  
- [x] Check for content containers using "edge-to-edge" or "full-width containers" and provide recommendation to use "normal" or "wide" for accessibility line-length.  
- [x] Check for extra whitespace at top of page or bottom, and recommend they update the section containers to "remove top spacing" or "remove bottom spacing"
- [x] Check for extra space between sections, sometimes sections stacked on top of each other will either not have enough space or extra space that limits the content flow. 
- [x] Check for spacing between sections.   Do layouts/sections run together or are they separated with subtle background colors with default spacing?  Recommend separating sections with a gray background color to help create content separation and visual hierarchy. 
  - From brand.uiowa.edu: "**Careful use of contrast establishes a clear visual hierarchy—drawing attention to the most important facets of your design and message. While our color palette creates immediate opportunity for contrast, the use of negative space, different sizes, shapes, and positioning of our visual elements also play a role in creating contrast and lending energy to layouts.**"

# Headings

- [x] Check for use of all lowercase in headings.  
- [x] Check for use of all caps in headings.
- [x] Check for consistency of heading usage on the page
  - Are they all centered, left aligned or is there a noticeable mismatch?  Recommend consistent usage for scanning the page.
  - How are headings placed?  Above content sections or within columns?  Recommend heading placement above sections for readability/scannablity of page. 
  - Are the headings all using different font styles and sizes?  If so, recommend consistent usage for readability. 
  - From brand.uiowa.edu: "**Use typography to create bold visuals with your headlines—keeping them short and direct, but compelling enough to draw a reader in.**"

**Comment by bspeare** [comment]
I added a checklist of criteria above to check when reviewing the site.   I'd be happy to pair with anyone to help clarify the criteria if anyone is interested in this work.

**Comment by joewhitsitt** [comment]
i'd sit in on this

**Comment by ABoston88** [comment]
Likewise.

**Comment by bspeare** [comment]
I'll start a huddle on this at 1 for those that are interested.   Let me know if that time works!

**Comment by ABoston88** [comment]
I have SiteNow training from 1 - 2:30. If we can delay a bit, cool. If not, no worries. I'll catch up when training ends if this is still happening.

**Comment by bspeare** [comment]
Good to know! How about 3?

**Comment by ABoston88** [comment]
That's perfect. Thanks!

**Comment by bspeare** [comment]
We looked through the site yesterday and made some general notes and suggestions.  After seeing how many notes we were making for each page, we decided it might be better to focus on providing some overall strategies to focus on and walk through a few of their page layouts with them at a future meeting.    

We are waiting to get a sample page that they consider "done" and we might create an alternative layout if we can get that example page before our next meeting with them on Monday.
